src/HueDeviceTypes.cpp

```diff
@@ -66,6 +66,13 @@ const std::set<std::string> getTemperatureLightTypes()
             "LTC003", "LTC004", "LTC011", "LTC012", "LTD001", "LTD002", "LFF001", "LTT001", "LDT001"};
     return c_TEMPERATURELIGHT_TYPES;
 }
+
+const std::set<std::string> getPlugTypes()
```

Comment:
Maybe call this getNonDimmableTypes(), so other lights that are not plugs can be put in here too.

Reply:
Should I add the new function to the end below `getTemperatureLightTypes` or between `getNoColorTypes` and `getTemperatureLightTypes` ?

Reply:
If you want to sort them, you could do gamut a,b,c, then temperature, then noColor (should probably be renamed to dimmable) and finally non dimmable.